cmd/registry.go

```diff
@@ -130,6 +176,22 @@ func printRegistries(regList []Registry) {
 	util.PrintTable(tableToPrint)
 }
 
+func applyOverrides(conf registries.Config) registries.Config {
+	if regOrg != "" {
+		conf.Org = regOrg
+	}
+	if regUrl != "" {
+		conf.URL = regUrl
+	}
+	if len(nsList) > 0 {
+		conf.Namespaces = nsList
+	}
+	if len(whitelist) > 0 {
+		conf.WhiteList = whitelist
+	}
+	return conf
+}
+
```

Comment:
So yeah, I think I would read the parameters into a registryConfig  and make this method look like:

```golang
func applyOverrides(conf *registries.Config, params registries.Config) {
    if params.Org != "" {                                               
        conf.Org = params.Org                                           
    }                                                                   
    if params.URL != "" {                                               
        conf.URL = params.URL                                           
    }                                                                   
    if len(params.Namespaces) > 0 {                                     
        conf.Namespaces = params.Namespaces                             
    }                                                                   
    if len(params.Whitelist) > 0 {                                      
        conf.WhiteList = params.Whitelist                               
    }
```